apps/aechannel/src/aesc_limits.erl

```diff
@@ -0,0 +1,90 @@
+-module(aesc_limits).
```

Comment:
This deserves documentation.

Reply:
I actually had documentation in the first version, that was unfortunately lost as I extracted myself from the mess caused by forgetting that I was in the middle of a rebase. :)

I will add some documentation again.